tests/data_context/test_data_context_datasources.py

```diff
@@ -0,0 +1,59 @@
+import pathlib
+from unittest import mock
+
+import pytest
+
+from great_expectations.data_context.data_context.base_data_context import (
+    BaseDataContext,
+)
+from great_expectations.data_context.store.ge_cloud_store_backend import (
+    GeCloudStoreBackend,
+)
+from great_expectations.data_context.store.inline_store_backend import (
+    InlineStoreBackend,
+)
+from great_expectations.data_context.types.base import DataContextConfig, GeCloudConfig
+
+
+@pytest.mark.integration
+@mock.patch(
+    "great_expectations.data_context.store.DatasourceStore.list_keys", return_value=[]
+)
```

Comment:
Not asking for any action now, but something could be useful in the future is moving away from ad-hoc mocking/patching to a few well crafted mock (or fake) pytest fixtures that get re-used.

Not a ton of a value add here, since the patch is very simple 😄 .

Reply:
I agree! Would love some pointers here moving forward